client/web/src/enterprise/batches/list/BatchChangeListPage.tsx

```diff
@@ -196,8 +197,8 @@ export const BatchChangeListPage: React.FunctionComponent<React.PropsWithChildre
                     </strong>
                 </LimitedAccessBanner>
             )}
-            <BatchChangesListIntro isLicensed={licenseAndUsageInfo?.batchChanges || licenseAndUsageInfo?.campaigns} />
-            {!isSourcegraphDotCom && (
+            <BatchChangesListIntro isLicensed={batchChangeLicensed || licenseAndUsageInfo?.campaigns} />
```

Comment:
You may or may know this already, but "Campaigns" is the original name of Batch Changes that existed on really old customer licenses. At this point, I think we can be fairly confident that all customers have been migrated to more modern licenses, and when we perform this same sort of gating on the backend, we no longer respect a `campaigns` license. So I'd say at this point
```suggestion
            <BatchChangesListIntro isLicensed={batchChangeLicensed} />
```
would be sufficient for us here!